### PR TITLE
Update the existing code with python 3.x

### PR DESCRIPTION
**Pull request by nishikeshKardak**

### CODE REVIEW

File: qtopic/pyqtopics.py [hunk 1]
```diff
@@ -72,46 +74,52 @@ def get_top_stories(topic):
             'title': title,
             'published': published
         }
-        return dict
+        return dict"""
```
Comment (tapaswenipathak) [inline qtopic/pyqtopics.py:77]: Why have you commented this method?

**Comment by tapaswenipathak** [comment]
Merging this, I see you opened a [ticket](https://github.com/tapaswenipathak/pyQTopic/issues/15).

**Comment by tapaswenipathak** [comment]
Closes https://github.com/tapaswenipathak/pyQTopic/issues/12.

File: qtopic/pyqtopics.py (hunk 1 above)
Comment (nishikeshKardak) [reply qtopic/pyqtopics.py:77]: I didn't find ```https://www.quora.com/topic/rss``` for any topic. That's why I have commented this method.

**Comment by nishikeshKardak** [comment]
Thanking You! for merging PL.

**Pull request merged**